src/main/java/com/lob/client/AsyncLobClient.java

```diff
@@ -495,10 +498,22 @@ private BoundRequestBuilder post(final String resourceUrl, final HasLobParams ha
         final SettableFuture<T> guavaFut = SettableFuture.create();
         try {
             final Optional<String> apiVersionOpt = Lob.getApiVersion();
+            String wrapperVersion;
+
             if (apiVersionOpt.isPresent()) {
                 request.addHeader(LobClient.LOB_VERSION_HEADER, apiVersionOpt.get());
             }
 
+            try {
+                Properties prop = new Properties();
+                prop.load(AsyncLobClient.class.getClassLoader().getResourceAsStream("project.properties"));
+                wrapperVersion = prop.getProperty("version");
```

Comment:
Close. It's fetching it from the project.properties new file that is created when the project is build. This is such that the version is always in sync with the maven version.